sql_generators/glean_usage/templates/baseline_clients_first_seen_v1.checks.sql

```diff
@@ -0,0 +1,19 @@
+{{ header }}
+
+{#
+   We use raw here b/c the first pass is rendered to create the checks.sql
+   files, and the second pass is rendering of the checks themselves.
+
+   For example, the header above is rendered for every checks file
+   when we create the checks file, when `bqetl generate glean_usage`
+   is called.
+
+   However the second part, where we render the check is_unique() below,
+   is rendered when we _run_ the check, during `bqetl query backfill`
+   (you can also run them locally with `bqetl check run`).
+#}
+{% raw -%}
+#warn
+{{ is_unique(["client_id"], where="submission_date = @submission_date") }}
```

Comment:
Shouldn't this check be checking the entire table for duplicates rather than a single partition? Otherwise, in combination with the daily table uniqueness check not sure this adds any value.